x-pack/plugins/cases/server/client/cases/create.test.ts

```diff
@@ -495,31 +495,11 @@ describe('create', () => {
       );
     });
 
-    it('should not throw an error and fill out missing customFields when they are undefined', async () => {
```

Comment:
I think we should keep this test and the new test you added ("should throw an error when required `customFields are undefined"). For this test, we need to make all fields in the configuration optional.